docs/faq.md

```diff
@@ -74,4 +75,57 @@ The property carbon.lock.type configuration specifies the type of lock to be acq
 ## How to resolve Abstract Method Error?
 In order to build CarbonData project it is necessary to specify the spark profile. The spark profile sets the Spark Version. You need to specify the ``spark version`` while using Maven to build project.
 
+## How Carbon will behave when execute insert operation in abnormal scenarios?
+Carbon support insert operetion, you can refer to the syntax mentioned in DML Operations on CarbonData.
+First, create a soucre table and load data into this created table. Create source table as follows:
+```
+CREATE TABLE source_table(
+id String,
+name String,
+city String)
+ROW FORMAT DELIMITED FIELDS TERMINATED BY ",";
+```
+Data inside source table as follows:
```

Comment:
Can remove the two sentences : "Create source table as follows:", "Data inside source table as follows:"